projects/js-packages/publicize-components/src/components/form/index.js

```diff
@@ -372,6 +376,14 @@ export default function PublicizeForm( {
 							? renderInstagramNotice()
 							: renderValidationNotice() ) }
 				</>
+			) : (
+				! shouldShowInstagramNotice && (
+					<PanelRow>
+						<ExternalLink href={ connectionsAdminUrl } target="_blank">
```

Comment:
I think `ExternalLink` already has `target="_blank"`
```suggestion
						<ExternalLink href={ connectionsAdminUrl }>
```

Reply:
Ah good point! I just restored the old code here.